.github/templates/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+*Descriptive Summary of your Bugfix, Feature, or Refactoring.*
+
+## List of Added or Changed Components
+- feature a added to module bar
+- bug b fixed in foo.rs
+
+## Open Questions
+- [ ] should I go with solution A or B?
+
+## Checklist
+- [ ] sensible git history (we are not too strict, but multiple "typo" or "fix" commits are not okay)
+- [ ] final rebase onto main before merging (if sensible)
```

Comment:
I think this can be dropped since it's already pretty much enforced by our repo config, requiring the PR to be up to date with latest main. This is easy for the maintainer to do with the rebase button in the UI, so no need for the author to do it unless there are conflicts.